inventory.py

```diff
@@ -257,6 +264,8 @@ def run_course_inventory() -> None:
     logger.info(f'Inserting {num_enrollment_records} enrollment records to DB')
     enrollment_df.to_sql('enrollment', db_creator_obj.engine, if_exists='append', index=False)
     logger.info(f'Inserted data into enrollment table in {db_creator_obj.db_name}')
+    logger.info(f"Inserting {num_canvas_usage_records} canvas_usage records to DB")
+    canvas_usage_df.to_sql('canvas_usage', db_creator_obj.engine, if_exists='append', index=False)
```

Comment:
Might want to add the `Wrote` and `Inserted` after log messages for consistency. It's pretty wordy, but it's useful if there's an insertion problem.

Reply:
Done